### PR TITLE
Amls 4911 - TCSP accessibility changes

### PR DESCRIPTION
**Pull request by matthewharris1**
Accessibility changes for the TCSP flow. Not to be merged until 5018 is complete
## How Has This Been Tested?
Manual
Unit

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/tcsp/another_tcsp_supervision.scala.html [hunk 1]
```diff
@@ -36,41 +35,33 @@
...
-                    value = "false",
-                    inline = true
-                )
-            }
+            @radio(
+                f = f("servicesOfAnotherTCSP"),
+                labelText = "lbl.yes",
+                value = "true",
+                target = "[data-reveal='mlr-ref-number']",
+                inline = true
```
Comment (drewmiley) [inline app/views/tcsp/another_tcsp_supervision.scala.html]: Remove inline

File: app/views/tcsp/complex_corp_structure_creation.scala.html [hunk 2]
```diff
@@ -32,29 +31,25 @@
...
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
-                )
-            }
+            @radio(
+                f = f("complexCorpStructureCreation"),
+                labelText = "lbl.yes",
+                value = "true",
+                inline = true
```
Comment (drewmiley) [inline app/views/tcsp/complex_corp_structure_creation.scala.html]: Remove inline

File: app/views/tcsp/only_off_the_shelf_comps_sold.scala.html [hunk 3]
```diff
@@ -32,29 +31,25 @@
...
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
-                )
-            }
+            @radio(
+                f = f("onlyOffTheShelfCompsSold"),
+                labelText = "lbl.yes",
+                value = "true",
+                inline = true
```
Comment (drewmiley) [inline app/views/tcsp/only_off_the_shelf_comps_sold.scala.html]: Remove inline

File: app/views/tcsp/services_of_another_tcsp.scala.html [hunk 4]
```diff
@@ -33,28 +32,25 @@
...
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
-                )
-            }
+            @radio(
+                f = f("servicesOfAnotherTCSP"),
+                labelText = "lbl.yes",
+                value = "true",
+                inline = true
```
Comment (drewmiley) [inline app/views/tcsp/services_of_another_tcsp.scala.html]: Remove inline

File: app/views/tcsp/another_tcsp_supervision.scala.html (hunk 1 above)
Comment (matthewharris1) [reply app/views/tcsp/another_tcsp_supervision.scala.html]: done

File: app/views/tcsp/complex_corp_structure_creation.scala.html (hunk 2 above)
Comment (matthewharris1) [reply app/views/tcsp/complex_corp_structure_creation.scala.html]: done

File: app/views/tcsp/only_off_the_shelf_comps_sold.scala.html (hunk 3 above)
Comment (matthewharris1) [reply app/views/tcsp/only_off_the_shelf_comps_sold.scala.html]: done

File: app/views/tcsp/services_of_another_tcsp.scala.html (hunk 4 above)
Comment (matthewharris1) [reply app/views/tcsp/services_of_another_tcsp.scala.html]: done

**Pull request merged**